source/_cookbook/SSL guide.md

```diff
@@ -0,0 +1,570 @@
+You may wish to set up other options for the http component at this point, these extra options are beyond the scope of this guide but can be found on the http component page here: [http](https://home-assistant.io/components/http/)
+
+Save the changes to configuration.yaml.  Restart HA.
+
+In step 3 we accessed our HA from the outside world with our DuckDNS URL and our port number.  We are going to use a slightly different URL this time.
+
+```text
+https://examplehome.duckdns.org
+```
+
+Note the S after http, and that no port number is added.  This is because https will use port 443 automatically, and we have already set up our port forward to redirect this request to our HA instance on port 8123.
+
+You should now be able to see your HA instance via your DuckDNS URL, and importantly note that your browser shows the connection as secure.
+
+You will now NO LONGER be able to access your HA via your old internal IP address in the way you previously have.  Your default way to access your HA instance, even from inside your house, is to use your DuckDNS URL.
+
+In cases where you need to access via the local network only (which should be few and far between) you can access it with the following URL (note the added S after http):
+
+```text
+https://YOUR-HA-IP:8123
+```
+
+...and accepting the browsers warning that you are connecting to an insecure site.  This warning occurs because your certificate expects your incoming connection to come via your DuckDNS URL.  It does not mean that your device has suddenly become insecure.
+
+Some cases such as this are where your router does not allow 'loopback' or where there is a problem with incoming connections due to technical failure.  In these cases you can still use your internal connection and ignore the warnings.
+
+If you were previously using a webapp on your phone/tablet to access your HA you should delete the old one and create a new one with the new address.  The old one will no longer work as it is not keyed to your new, secure URL.  Instructions for creating your new webapp can be found here: 
+
+```text
+https://home-assistant.io/docs/frontend/mobile/
+```
+
+All done?  Accessing your HA from across the world with your DuckDNS URL and a lovely secure logo on your browser?  Ace!  Now lets clean up our port forwards so that we are only exposing the parts of our network that are absolutely neccesary to the outside world....
+
+#### {% linkable_title 6 - Clean up port forwards %}
+
+In step 2 we created a port forwarding rule called ha_test.  This opens port 8123 to the world, and is no longer neccessary.
+
+Go to your router's configuration pages and delete the ha_test rule.
+
+You chould now have two rules in relation to HA for your port forwards, named:
+
+```text
+ha_ssl and ha_letsencrypt
+```
+
+If you have any more for HA you should delete them now.  If you only have ha_ssl this is probably because during step 4 you had to use port 443 instead of port 80, so we deleted the rule during step 5.

```

Comment:
Rules, is it really that unclear? I think when it is 'presented' and formatted it will look right, but if it doesn't I'll reword it. Not sure what to though?